public-ui/src/app/data-browser/chart/chart.component.ts

```diff
@@ -101,7 +101,12 @@ export class ChartComponent implements OnChanges {
             style: this.isGenderIdentityAnalysis()
                 ? this.dbc.GI_DATA_LABEL_STYLE : this.dbc.DATA_LABEL_STYLE,
             distance: this.isGenderIdentityAnalysis() ? 3 : -30,
-            format: '{point.name} {point.percentage:.0f}%',
+            formatter: function () {
```

Comment:
nit: arrow function is preferred

Reply:
This unfortunately seems to be tough as the arrow function gets the whole context of the document and unlike chart's context in this function. So, leaving it as it is for now.

Reply:
Ah I see - thanks for checking.